specification/keyvault/Security.KeyVault.Common/common.tsp

```diff
@@ -0,0 +1,35 @@
+/**
+ * The key vault server error.
+ */
+model Error {
+  /**
+   * The error code.
+   */
+  @visibility("read")
+  code?: string;
```

Comment:
Under what circumstances is this actually not returned?

Reply:
@heaths may know of a particular scenario, but I also wonder if this being [originally](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.5/common.json#L52) marked as optional was an oversight. Or, since the error itself is nullable, maybe that was reasoning for the properties being optional?

Reply:
A "parent" (envelope etc.) being optional doesn't mean that the constituent parts are also optional. I suspect this is because of the `@visibility("read")` combined with the fact that openapi doesn't allow read-only required properties. But if the model is only every used in/as a return type, marking the property as read-only is superfluous/not needed.